README.md

```diff
@@ -945,7 +945,7 @@ Using the translation key `fields.field-name.label` will return different values
 
 # HOF Components
 
-## Date Component
+# Date Component
```

Comment:
shouldn't this be a sub heading rather than a main heading?

Reply:
👍🏾 I have amended this now, All the other components were also using the main heading levels so I have made them subheadings and also adjusted the levels of their corresponding subheadings